spring-security-modules/spring-security-oidc/src/main/java/com/baeldung/openid/oidc/jwtauthorities/config/SecurityConfig.java

```diff
@@ -64,12 +64,9 @@ public Converter<Jwt,AbstractAuthenticationToken> customJwtAuthenticationConvert
     @Bean
     SecurityFilterChain customJwtSecurityChain(HttpSecurity http) throws Exception {
         // @formatter:off
-        return http.oauth2ResourceServer(oauth2 -> {
-            oauth2.jwt()
-              .jwtAuthenticationConverter(customJwtAuthenticationConverter(accountService));
-        })
+        return http.oauth2ResourceServer(oauth2 -> oauth2.jwt(jwtConfigurer -> jwtConfigurer
+                                .jwtAuthenticationConverter(customJwtAuthenticationConverter(accountService))))
```

Comment:
Can you convert this to three lines please and fix the indentations.

Reply:
Fixed